gp-conditional-pricing/gpcp-product-pricing-labels.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Gravity Perks // GP Conditional Pricing // Display Price Labels (JS)
+ * http://gravitywiz.com/documentation/gravity-forms-conditional-pricing/
+ *
+ * This snippet requires the PHP counterpart gpcp-product-pricing-labels.php
+ */
+( function( $ ) {
+	function update_price_lables() {
+		$( 'label[data-gpcp-template], option[data-gpcp-template]' ).each( function() {
+			var $priceElem = $( this ).is( 'option' ) ? $( this ) : $( this ).siblings( 'input' ),
+				price = gformFormatMoney( $priceElem.val().split( '|' )[1] ),
+				template = $( this ).attr( 'data-gpcp-template' );
```

Comment:
Can we update this so each variable has a `var` and each line ends with a semi-colon?

Reply:
Sure. Fixed in 053f88d.